### PR TITLE
update to support velocity report header

### PR DESCRIPTION
**Pull request by TakaHoribe**
Signed-off-by: Takamasa Horibe <horibe.takamasa@gmail.com>

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by TakaHoribe** [comment]
merge with https://github.com/tier4/autoware_auto_msgs/pull/23

File: vehicle/ssc_interface/src/ssc_interface.cpp [hunk 1]
```diff
@@ -166,6 +166,8 @@ void SSCInterface::callbackFromSSCFeedbacks(
   const pacmod3_msgs::msg::WheelSpeedRpt::ConstSharedPtr msg_wheel_speed,
   const pacmod3_msgs::msg::SystemRptFloat::ConstSharedPtr msg_steering_wheel)
 {
+  (void)msg_throttle;
```
Comment (tkimura4) [inline vehicle/ssc_interface/src/ssc_interface.cpp:169]: (細かいですが…)
[[maybe_unused]]使ったらどうでしょうか

File: simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp [hunk 2]
```diff
@@ -398,7 +398,8 @@ geometry_msgs::msg::TransformStamped SimplePlanningSimulator::get_transform_msg(
 void SimplePlanningSimulator::publish_velocity(const VelocityReport & velocity)
 {
   VelocityReport msg = velocity;
-  msg.stamp = get_clock()->now();
+  msg.header.stamp = get_clock()->now();
+  msg.header.frame_id = "base_link";
```
Comment (tkimura4) [inline simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp:402]:
```suggestion
  msg.header.frame_id = simulated_frame_id_;
```

**Comment by TakaHoribe** [comment]
@tkimura4 done!

**Review by tkimura4 — APPROVED** [review]
LGTM

**Comment by tkimura4** [comment]
(注: これマージされないとrviz落ちる状態なので, Build and test for PR落ちたら頑張ってください)

**Pull request merged**